### ISSUE TITLE
Question about MIDI over USB to computer

### ISSUE THREAD

**Issue by spiderWAW** [YuuichiAkagawa/USBH_MIDI#11]
Today if we compare your project with commercial one like discontinued  iConnectivity iConnectMIDI just only one thing is missing: parallel bidirectional connection with computer over cable used to programing Arduino. Is possible to ad this functionality to your project? 

USB MIDI Host IN > MIDI 5 Out and over USB to computer (as MIDI IN_)
USB MIDI Host Out and over USB from computer (as MIDI Out) > USB MIDI Host In and 5 DIN Out 
USB MIDI Host Out and 5 DIN In > Computer over USB ie. MIDI IN 

BTW: Fishman TriplePlay is supported (IN/OUT); Tutorial based on your library http://www.vguitarforums.com/smf/index.php?topic=9681.msg69488#msg69488

**Comment by lokkiikkol** [comment]
yes this should be trivial with an arduino leonardo or micro and an usb shield. i will attempt this soon!

**Comment by lokkiikkol** [comment]
this also is trivial with an arduino uno. you can reprogram the 16u2 used as usb-to-serial to be used as usb-to-midi. that way you get exactly what you want, see here:

http://morecatlab.akiba.coocan.jp/lab/index.php/aruino/midi-firmware-for-arduino-uno-moco/?lang=en
